src/backup.ts

```diff
@@ -16,7 +16,19 @@ import {PreciseDate} from '@google-cloud/precise-date';
 import {promisifyAll} from '@google-cloud/promisify';
 import snakeCase = require('lodash.snakecase');
 import {google} from '../protos/protos';
-import {Bigtable, Cluster, Table} from './';
+import {
+  Bigtable,
+  Cluster,
+  GetIamPolicyCallback,
+  GetIamPolicyOptions,
+  GetIamPolicyResponse,
+  Policy,
+  SetIamPolicyCallback,
+  SetIamPolicyResponse,
+  TestIamPermissionsCallback,
+  TestIamPermissionsResponse,
+} from './';
+import {Table} from '../src/table';
```

Comment:
Why are we importing Table from a different location now? I didn't notice any refactoring to move that around, but I may have missed it.

Reply:
`Table.prototype.<method>` is used for `getIamPolicy`, `setIamPolicy` and `testIamPermissions` methods to be bound with `backup` object and at run time `Table` is undefined if not imported directly from `'./table'`